src/sales/checkout-payment-failed-emails.md

```diff
@@ -3,7 +3,7 @@ title: Payment Failed Email
 group: marketing
 ---
 
```

Comment:
I need some clarification for "an admin" -- Is it a Magento Admin user, and is the email configured somewhere?